utils/db.ts

```diff
@@ -29,6 +28,36 @@ async function getValues<T>(
   return values;
 }
 
+async function getMany<T>(
+  keys: Deno.KvKey[],
+): Promise<Deno.KvEntryMaybe<T>[]> {
+  const res: Deno.KvEntryMaybe<T>[] = [];
+  for (const batch of batches(keys, 10)) {
+    res.push(...(await kv.getMany<T[]>(batch)));
+  }
+  return res;
+}
+
+function* batches<T>(array: T[], size: number): Generator<T[], void> {
```

Comment:
Have you checked whether the Standard Library has something like this in either `std/collections` or `std/async` that we could use instead? If not, all good.

Reply:
nice, didn't knew about `std/collections`!
refactored the logic to make use of `chunk`.